build-config/make/lvm2.make

```diff
@@ -1,6 +1,7 @@
 #-------------------------------------------------------------------------------
```

Comment:
I uploaded lvm-2_02_155 to the ONIE mirror:
```
monster-15:/tmp/goo$ wget http://mirror.opencompute.org/onie/lvm2-2_02_155.tar.xz
--2016-08-01 08:42:59--  http://mirror.opencompute.org/onie/lvm2-2_02_155.tar.xz
Resolving mirror.opencompute.org (mirror.opencompute.org)... 23.253.235.92
Connecting to mirror.opencompute.org (mirror.opencompute.org)|23.253.235.92|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1469316 (1.4M) [application/x-xz]
Saving to: `lvm2-2_02_155.tar.xz'

100%[====================================================================================>] 1,469,316   3.38M/s   in 0.4s    

2016-08-01 08:42:59 (3.38 MB/s) - `lvm2-2_02_155.tar.xz' saved [1469316/1469316]
```

The sha1sum is:
```
monster-15:/tmp/goo$ sha1sum lvm2-2_02_155.tar.xz 
e1dd87307a0127e3f1469408051fed2a6add2922  lvm2-2_02_155.tar.xz
```

Let me know if you agree with the version and sha1sum.